submariner-operator/submariner-metrics-redesign.md

```diff
@@ -0,0 +1,163 @@
+# Submariner metrics redesign
+
+Related Issue:
+[Exporting Submariner metrics without needing to open ports](https://github.com/submariner-io/submariner/issues/1944)
+
+## Summary
+
+Submariner Gateway and Globanet pods run with HostNetworking. To access metrics exported by these two we need to open ports
+in the infra firewall. Current design has a limitation where these ports might be in use by some other service,
+causing a conflict and metrics becoming inaccessible. With the move towards minimizing changes needed in cloud infra for
+submariner, there is a use case to export metrics in a way that doesn't require any of these ports to be opened.
+
+## Proposal
+
+This enhancement proposes a redesign of existing metrics for Submariner Gateway and Globalnet by adding a new metrics proxy
+service. Pods running on the same node can access ports without requiring ports to be opened in the cloud infra. Any queries
+for these metrics will land on the proxy which will always run on the same node.
+
+This proposal also recommends changing the port used by metrics http servers in gateway and globalnet pods to a free range
+port to avoid problem of port being already in use on host network.
```

Comment:
```suggestion
This proposal also recommends changing the ports used by the metrics HTTP servers in the Gateway and Globalnet pods to a free range
port to avoid conflicts with ports already in use on the host network.
```